api/src/main/java/org/pmiops/workbench/mandrill/MandrillService.java

```diff
@@ -0,0 +1,14 @@
+package org.pmiops.workbench.mandrill;
+
+import org.pmiops.workbench.mandrill.model.MandrillMessage;
+import org.pmiops.workbench.mandrill.model.MandrillMessageStatuses;
+
+/**
+ * Encapsulate mandrill API interaction details and provide a simple/mockable interface
+ * for internal use.
+ */
+public interface MandrillService {
```

Comment:
I don't think we actually need a new service for this that's mandrill-specific. All of this code could be added to the current `MailService` and `MailServiceImpl` classes.